optuna/storages/rdb/storage.py

```diff
@@ -37,19 +37,23 @@
 class RDBStorage(BaseStorage):
     """Storage class for RDB backend.
 
-    This class is not supposed to be directly accessed by library users.
+    Note that library users can instantiate this class, but the fields and methods
```

Comment:
I believe that it could be helpful to add an example here in the docs how it can be constructed and passed to `create_study`, unless there is an example already in which case we could add a reference? It's also perfectly fine to discuss in a separate PR.

Reply:
Nice idea! Because there is no existing example that using `RDBStorage`, I added an example code to the doc (https://github.com/pfnet/optuna/pull/689/commits/ebe9485dd180138a8dc975a7b7b0938805f44f51).

The following image is the rendered doc:
![image](https://user-images.githubusercontent.com/181413/68743897-c74e0080-0636-11ea-9a9f-13ccef8edd30.png)

Please let me know if you have any thoughts about this example.

Reply:
That's awesome. I'll have a look!